packages/apple-targets/src/colorset/customColorFromCSS.ts

```diff
@@ -11,7 +10,7 @@ export function customColorFromCSS(color: string): {
   /** @example `1` */
   alpha: number;
 } {
-  let colorInt = normalizeColor(color);
+  let colorInt = normalizeColor(color)!;
```

Comment:
instead of assuming this is not null, there could be a safe fallback, what do you think @EvanBacon ?